core/db/migrate/20220201103922_add_first_name_and_last_name_to_spree_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddFirstNameAndLastNameToSpreeUsers < ActiveRecord::Migration[5.2]
+  def change
+    if Spree.user_class.present?
+      add_column :spree_users, :first_name, :string unless column_exists?(:spree_users, :first_name)
```

Comment:
```suggestion
      add_column Spree.user_class.table_name, :first_name, :string unless column_exists?(:spree_users, :first_name)
```

the table name can be different than spree_users